upgrade notes: correct dedupe command lines

valentijnscholten opened the pull request:
Some small fixes here mainly around quoting.

dogboat reviewed (commented):
Are the backticks still desired? The newline/indents already get a code/paragraph that stands out. Preview:

![Screenshot 2025-03-13 at 3 45 42 PM](https://github.com/user-attachments/assets/362b845d-ceab-416f-8ab9-1604877e1bb5)

valentijnscholten commented:
> Are the backticks still desired? The newline/indents already get a code/paragraph that stands out. Preview:

Good point, a shame github doesn't render the preview for us in the PR :-)

Removed. Also changed `2.44.x` to `2.44.0`.
